presto-iceberg/src/test/java/com/facebook/presto/iceberg/hivemode/TestHiveIcebergOrcMetricsCollection.java

```diff
@@ -11,9 +11,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.iceberg;
+package com.facebook.presto.iceberg.hivemode;
```

Comment:
I noticed that for the native mode, there's no corresponding `TestNativeIcebergOrcMetricsCollection` file. Does the native mode support it?